readyplayerme/rpm-unity-sdk-core
# [SDK-806] - Add user auth

## [SDK-806](https://ready-player-me.atlassian.net/browse/SDK-806)

## Description

- Fix an issue, where when the avatar is deleted, it is not replaced with a new draft avatar
- save user auth state

## How to Test

-   Check, that when you start and stop the creator, the state is saved

[SDK-806]: https://ready-player-me.atlassian.net/browse/SDK-806?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
